src/edu/washington/escience/myria/coordinator/MasterCatalog.java

```diff
@@ -1728,16 +1733,17 @@ public void registerFunction(
                 protected Void job(final SQLiteConnection sqliteConnection)
                     throws CatalogException, SQLiteException {
                   try {
-                    deleteFunctionIfExists(sqliteConnection, functionName, false);
+                    deleteFunctionIfExists(sqliteConnection, functionName);
```

Comment:
Is this `DELETE` necessary if we have overwrite semantics? Since the `INSERT` will always be done in a single transaction, I don't see why it's necessary if we can overwrite.